pkg/packagekit/updates.jsx

```diff
@@ -449,57 +555,259 @@ class ApplyUpdates extends React.Component {
     }
 }
 
-const AskRestart = ({ onIgnore, onRestart, history }) => <>
-    <EmptyStatePanel icon={RebootingIcon}
-                     title={ _("Restart recommended") }
-                     paragraph={ _("Updated packages may require a restart to take effect.") }
-                     action={ _("Restart now") }
-                     onAction={ onRestart}
-                     secondary={ <Button variant="link" onClick={onIgnore}>{_("Ignore")}</Button> } />
-
-    <div className="flow-list-blank-slate">
-        <Expander title={_("Package information")}>
-            <PackageList packages={history[0]} />
-        </Expander>
-    </div>
-</>;
-
-const StatusCard = ({ updates, highestSeverity, timeSinceRefresh }) => {
-    const numUpdates = Object.keys(updates).length;
-    const numSecurity = count_security_updates(updates);
-    let stateStr;
-
-    if (numUpdates == 0)
-        stateStr = STATE_HEADINGS.uptodate;
-    else if (numUpdates == numSecurity)
-        stateStr = cockpit.ngettext("$1 security fix", "$1 security fixes", numSecurity);
-    else {
-        stateStr = cockpit.ngettext("$0 update", "$0 updates", numUpdates);
-        if (numSecurity > 0)
-            stateStr += cockpit.ngettext(", including $1 security fix", ", including $1 security fixes", numSecurity);
+const UpdateSuccess = ({ onIgnore, openServiceRestartDialog, openRebootDialog, restart, manual, reboot, tracerAvailable, history }) => {
+    if (!tracerAvailable) {
+        return (<>
+            <EmptyStatePanel icon={RebootingIcon}
+                             title={ _("Update was successful") }
+                             paragraph={ _("Updated packages may require a restart to take effect.") }
+                             secondary={
+                                 <>
+                                     <Button id="reboot-system" variant="primary" onClick={openRebootDialog}>{_("Reboot system...")}</Button>
```

Comment:
Please keep "Restart now", to not unnecessarily break translations. Also, "reboot" is a technical term, and does not match the "require a restart" from the paragraph above.

Reply:
The idea here was to make a bigger difference between "system restart" and "restart service" since restarting a service has been introduced to updates page. Reboot doesn't seem that technical to me

Reply:
Discussed on IRC. Let's keep "reboot" then. I'll resolve the other comments about that.